src/AzurePipelinesToGitHubActionsConverter.Core/Conversion/VariablesProcessing.cs

```diff
@@ -14,6 +14,21 @@ public class VariablesProcessing
     {
         private readonly bool _verbose;
         public List<string> VariableList;
+
+        private Dictionary<string, string> SystemVarMapping = new Dictionary<string, string>()
+        {
+            { "Agent.WorkFolder", "runner.workspace" },
+            { "Build.BuildId", "github.run_id" },
+            { "Build.BuildNumber", "github.run_number" },
+            { "Build.DefinitionName", "github.workflow" },
+            { "Build.SourcesDirectory", "github.workspace" }, // workspace is shared work folder, may need to create subfolder(s)
+            { "Build.ArtifactStagingDirectory", "github.workspace" }, // workspace is shared work folder, may need to create subfolder(s)
+            { "Build.SourceBranch", "github.ref" },
+            { "Build.SourceBranchName", "github.ref" }, // not exact mapping "main" vs "refs/heads/main"
+            { "Build.RepositoryName", "github.repository" }, // not exact mapping "gitutil" vs "naterickard/gitutil"
```

Comment:
Make the comment here more generic? i.e. [repo name] vs. [account/org name]/[repo name]?

Reply:
...was trying to give specific examples for some of these mappings, but I can see not wanting my username in there. Updated.